spectrocloud/resource_cluster_vsphere.go

```diff
@@ -219,7 +219,7 @@ func resourceClusterVsphere() *schema.Resource {
 			"cluster_rbac_binding": schemas.ClusterRbacBindingSchema(),
 			"namespaces":           schemas.ClusterNamespacesSchema(),
 			"host_config":          schemas.ClusterHostConfigSchema(),
-			"location_config":      schemas.ClusterLocationSchema(),
+			"location_config":      schemas.ClusterLocationSchemaComputed(),
```

Comment:
why do we need computed schema for VMware, shouldn’t we be able actually set VMware location?

Reply:
get back to schemas.ClusterLocationSchema() and handled in cluster_location _config